src/emoji.js

```diff
@@ -31,7 +30,8 @@ function getSkinTone () {
 // Note: In JXA, console.log writes to stderr instead of stdout
 function run (argv) {
   const query = argv[0]
-  const found = search(query, skinTone, pasteByDefault)
+  const operationType = getenv('snippetapp') ? 'snippet' : (argv[1] || 'clipboard')
```

Comment:
A bit awkward, but I think it's okay. Will need to split this if we add in extra shenanigans.

Reply:
Can we add a comment here to explain what `argv[1]` is and why it would take precedence to the `clipboard` string?